webapp/src/plugins/webfontloader.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * plugins/webfontloader.ts
+ *
+ * webfontloader documentation: https://github.com/typekit/webfontloader
+ */
+
+export async function loadFonts() {
+  const webFontLoader = await import(
+    /* webpackChunkName: "webfontloader" */ 'webfontloader'
+  );
+
+  webFontLoader.load({
+    google: {
+      families: ['Roboto:100,300,400,500,700,900&display=swap'],
+    },
+  });
+}
```

Comment:
We don't want to use webfontloader, I'd prefer directly importing local fonts using the `fontsource` packages.